src/pages/nft/class/_classId/index.vue

```diff
@@ -581,11 +602,13 @@ export default {
       );
       return this.handleCollect();
     },
-    handleCollectFromEdition(selectedValue) {
+
+    handleCollectFromEdition(selectedValue, actionType) {
```

Comment:
I think it is cleaner to separate the methods without adding extra param for `actionType`
```js
handleCollectFromEdition(selectedValue) {
  ...
},

handleCollectFromEditionSelector(selectedValue) {
  this.handleCollectFromEdition(selectedValue);
  logTrackerEvent(....);
},
handleCollectFromEditionTable(selectedValue) {
  this.handleCollectFromEdition(selectedValue);
  logTrackerEvent(....);
}
```